.github/workflows/node.js.yml

```diff
@@ -25,6 +40,11 @@ jobs:
       run: npm run typecheck
     - name: Run Linting Rules
       run: npm run lint
+    - name: Create .env file
+      run: |
+        echo "${{ secrets.TEST_ENV }}" > ./apps/server/.env.test
```

Comment:
This usage is fine but remember we are just testing. So you can always add your test env variables inside this yaml file. Just a reminder. Otherwise this usage is more secure and robust.